src/Support/OperationExtensions/ResponderExtension.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Dedoc\Scramble\Support\OperationExtensions;
+
+use Dedoc\Scramble\Extensions\OperationExtension;
+use Dedoc\Scramble\Support\Generator\Combined\AnyOf;
+use Dedoc\Scramble\Support\Generator\Operation;
+use Dedoc\Scramble\Support\Generator\Response;
+use Dedoc\Scramble\Support\Generator\Schema;
+use Dedoc\Scramble\Support\Generator\Types as OpenApiTypes;
+use Dedoc\Scramble\Support\RouteInfo;
+use Dedoc\Scramble\Support\Type\Generic;
+use Illuminate\Support\Collection;
+use PhpParser\Node\Stmt\ClassMethod;
+use PhpParser\Node\Stmt\Return_;
+
+class ResponderExtension extends OperationExtension
+{
+    public function handle(Operation $operation, RouteInfo $routeInfo)
```

Comment:
What's the return type?

Reply:
it doesn't return anything

Reply:
```suggestion
    public function handle(Operation $operation, RouteInfo $routeInfo): void
```
Let's add the `void` return type then.